src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseFileUploadsSection.tsx

```diff
@@ -36,6 +36,8 @@ const errorMappings = [
       FILE_TYPE_INVALID: 'Choose a file of an allowed format',
       FILENAME_CANNOT_CONTAIN_SPACES_OR_SPECIAL_CHARACTERS:
         'Filename cannot contain spaces or special characters',
+      FILE_UPLOAD_NAME_CANNOT_CONTAIN_SPECIAL_CHARACTERS:
```

Comment:
This is targetting the wrong form field. It should be mapped against `name`.

Reply:
Yep - well spotted :) - done